esi_leap/db/sqlalchemy/models.py

```diff
@@ -77,7 +77,7 @@ class Contract(Base):
     offer_uuid = Column(String(36),
                         ForeignKey('offers.uuid'),
                         nullable=False)
-
+    marketplace_offer_contract_relationship_id = Column(String(36), nullable=False)
```

Comment:
You'll want to move this up a few lines, maybe just below the project_id

Reply:
Done